habitat-baselines/habitat_baselines/articulated_agent_controllers/humanoid_rearrange_controller.py

```diff
@@ -161,9 +187,10 @@ def get_walk_pose(
                 new_angle *= np.pi / 180
                 did_rotate = True
             else:
-                new_angle = curr_angle * np.pi / 180
+                new_angle = new_angle * np.pi / 180
 
             forward_V = mn.Vector3(np.sin(new_angle), 0, np.cos(new_angle))
+            new_angle = new_angle * 180 / np.pi
```

Comment:
Why are we setting a value for new_angle here? It does not look like it is used anywhere later in the code.

Reply:
Addressed in last commit